### PR TITLE
346 - Sawmill production

### PR DESCRIPTION
**Pull request by ArchmageInc**
This determines the production and allocation of sawmills.

### CODE REVIEW

**Comment by ArchmageInc** [comment]
yay

**Comment by allensb** [comment]
yay

File: laws.md [hunk 1]
```diff
@@ -137,3 +137,5 @@ Where N is the value of the population at the time of the calculation and H is t
 **344.** **INITIAL PLAYER POINTS:** New Players, upon their first creation, will be awarded an initial point value set by a roll (1d12 + 12).
 
 **345.** **SAWMILL PURCHASES:** All players with a village may purchase sawmills for one hundred (100) points each. Sawmills produce Lumber.
+
+**346.** **SAWMILL PRODUCTION:** Each Sawmill will produce 1d12+3 lumber per week as long as four (6) villagers per sawmill are allocated to the logging endevor.
```
Comment (jimmyhmiller) [inline laws.md]: > four (6)

What??

**Comment by dilbertKocik** [comment]
You know you could simply factor in the number of workers as a multiplier on the productivity. It doesn't make sense that a sawmill operated by 3 people wouldn't produce anything. This applies to farming as well but I didn't think of it then.

File: laws.md (hunk 1 above)
Comment (ArchmageInc) [inline laws.md]: good point. Fixed

**Comment by ArchmageInc** [comment]
@dilbert88 Yes, that is a good idea and adds much needed complexity. I'll look forward to the laws & bot modifications in the future. :)

**Comment by dilbertKocik** [comment]
Adds complexity? Unless I am very much mistaken you should be able to remove the floor function from getActiveSawMills and the rest will work.

**Comment by jimmyhmiller** [comment]
yay

**Comment by dilbertKocik** [comment]
yay

**Pull request merged**